pytorch_lightning/utilities/parsing.py

```diff
@@ -79,23 +79,46 @@ def clean_namespace(hparams):
         del hparams_dict[k]
 
 
+def parse_class_init_keys(cls) -> Tuple[str, str, str]:
+    """Parse key words for standard self, *args and **kwargs
+
+    >>> class Model():
+    ...     def __init__(self, hparams, *my_args, anykw=42, **my_kwargs):
+    ...         pass
+    >>> parse_class_init_keys(Model)
+    ('self', 'my_args', 'my_kwargs')
+    """
+    init_parameters = inspect.signature(cls.__init__).parameters
+    # docs claims the params are always ordered
+    # https://docs.python.org/3/library/inspect.html#inspect.Signature.parameters
+    init_params = list(init_parameters.values())
+    # self is always first
+    n_self = init_params[0].name
+
+    def _get_first_if_any(params, str_type):
+        all_vars = [p for p in params if str(p.kind) == str_type]
+        return all_vars[0].name if all_vars else None
```

Comment:
This is very minor so feel free to ignore it but, if you are only interested in the first value, why not just use a loop?

```suggestion
        for p in params:
            if str(p.kind) == str_type:
                return p.name
```

Reply:
ok turned to be a bug here...